pkg/services/object/get/assemble.go

```diff
@@ -203,7 +203,9 @@ func (exec *execCtx) buildChainInReverse(prev oid.ID) ([]oid.ID, []objectSDK.Ran
 
 	// fill the chain end-to-start
 	for withPrev {
-		if exec.curOff < from {
+		// check that only for "range" requests,
+		// for `GET` it stops via the false `withPrev`
+		if seekRng != nil && exec.curOff <= from {
```

Comment:
Was there a bug previously?

Reply:
yes, if a child obj's boundary is equal to the original `Range`'s offset, a node does another useless `[X, 0]` `Range` request

fixed by `<` => `<=` change